packages/hawtio/src/plugins/shared/connect-service.ts

```diff
@@ -242,10 +242,27 @@ class ConnectService implements IConnectService {
    * Redirect to the URL specified in the query parameter {@link PARAM_KEY_REDIRECT}.
    */
   redirect() {
-    const url = new URL(window.location.href)
-    const redirect = url.searchParams.get(PARAM_KEY_REDIRECT) ?? hawtio.getBasePath() ?? '/'
+    const url = new URL(decodeURI(window.location.href))
```

Comment:
This one as well. Why do we need decode here?

Reply:
because we are sending encoded URL from the jolokia-service: https://github.com/mmelko/hawtio-next/blob/600af730d342d388244afa01973562a92a72188a/packages/hawtio/src/plugins/shared/jolokia-service.ts#L326